### PR TITLE
Updated Docs for v3 (Part 2)

### PR DESCRIPTION
**Pull request by carltongibson**
More incoming as I work through them... 🚀

Refs #1527

### CODE REVIEW

File: docs/topics/routing.rst [hunk 1]
```diff
@@ -98,15 +101,16 @@ URLRouter
...
+single argument, a list of Django URL objects (either ``path()`` or
+``re_path()``):
 
 .. code-block:: python
 
     URLRouter([
-        url(r"^longpoll/$", LongPollConsumer),
-        url(r"^notifications/(?P<stream>\w+)/$", LongPollConsumer),
-        url(r"", AsgiHandler),
+        re_path(r"^longpoll/$", LongPollConsumer),
```
Comment (carltongibson) [inline docs/topics/routing.rst:111]: This needs an `as_asgi()`

File: docs/topics/routing.rst [hunk 2]
```diff
@@ -23,38 +23,46 @@ more protocol-specific routing underneath there.
...
-        "aprs": APRSNewsConsumer,
-
     })
 
+.. note::
+  We call the ``as_asgi()`` classmethod when routing our consumers. This
+  returns an ASGI wrapper application that will instantiate a new consumer
+  instance for each connection or scope. This is similar to Django's
+  ``as_view()``, which plays the same role for per-request instances of
+  class-bases views.
```
Comment (smithdc1) [inline docs/topics/routing.rst:64]: Bases -> based?

File: docs/topics/sessions.rst [hunk 3]
```diff
@@ -32,7 +32,7 @@ in your ``routing.py``:
 
         "websocket": SessionMiddlewareStack(
             URLRouter([
-                url(r"^front(end)/$", consumers.AsyncChatConsumer),
+                url(r"^front(end)/$", consumers.AsyncChatConsumer.as_asgi()),
```
Comment (smithdc1) [inline docs/topics/sessions.rst:35]: While we're here. Url->re_path?

I'd be happy to take an action to conduct a general review for URL() if that'd help.

Comment (carltongibson) [reply docs/topics/sessions.rst:35]: Yes, these should all be `re_path`. (Not `path` if nested in a middleware...) Thanks!

**Pull request merged**